content/learn/_index.es.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Aprender"
+menu_title: "Aprender"
+mobile_menu_title: "Aprender"
+toc: false
+layout: single
+---
+
+# Aprende
+Esta sección incluye recursos útiles que te llevan de no saber nada a entender la filosofía del lenguaje.
+
+## Documentación
+{{% learn_docs %}}
+
+## Introdución
+Estos son recursos de introducción a Zig orientados a programadores con diferentes niveles de experiencia.
+
+- [Revisión a fondo](overview/)
+Esta es una vista a fondo de las características de Zig desde la perspectiva de un programador de sistemas.
+- [¿Por qué Zig, cuando ya existen C++, D, y Rust?](why_zig_rust_d_cpp/)
+Una introducción a Zig para desarrolladores de C++, D y Rust.
+- [Ejemplos de código](samples/)
+Una lista de ejemplos que muestran como luce el código Zig.
+- [Herramientas](tools/)
+Una lista de herramientas útiles al escribir código Zig.
+
+## Primeros pasos
+Si estas listo para comenzar a programar con Zig, esta guía te ayudará a poner en marcha el ambiente necesario.
```

Comment:
I think something like `configurar un entorno de programación` is possibly clearer here

nit: `estás`

Reply:
well... both sound correct to me